api/WindowEventHandlers.json

```diff
@@ -331,10 +331,10 @@
               "version_added": "24"
             },
             "safari": {
-              "version_added": null
+              "version_added": "10.1"
```

Comment:
Can you also update the languagechange_event entry to match? Or search for "languagechange" to find everything related.